### PR TITLE
Epilogue Message Accounts For Your Followers

### PR DESCRIPTION
**Pull request by bombasticSlacks**
#### Summary
None

#### Purpose of change
Death Epilogue should mention/allude to all your followers are dead or have given up on you as well.

#### Describe the solution
mention followers in the epilogue.

#### Describe alternatives you've considered
I don't know what the difference between the default and follower epilogue is so I did em both.

#### Testing
Haven't tested yet looking for feedback on the content.

### CODE REVIEW

**Comment by Maleclypse** [comment]
Don't each of your followers get individual epilogues?

**Comment by bombasticSlacks** [comment]
My thought is that anyone you don't take over and play as after death has abandoned you as a follower. The "you are forgotten" just seemed odd now that you can be on character 4 of the same play through. So signifying that this is where the story of you *and your followers* ends seemed valuable.

**Comment by Maleclypse** [comment]
> My thought is that anyone you don't take over and play as after death has abandoned you as a follower. The "you are forgotten" just seemed odd now that you can be on character 4 of the same play through. So signifying that this is where the story of you _and your followers_ ends seemed valuable.

That’s an excellent reason for this change.

File: data/json/snippets/epilogue_factions.json [hunk 1]
```diff
@@ -3,10 +3,13 @@
     "type": "snippet",
     "category": "epilogue_factions",
     "text": [
-      { "id": "epilogue_faction_default", "text": "    You are forgotten among the billions lost in the Cataclysm…" },
+      {
+        "id": "epilogue_faction_default",
+        "text": "    You and everyone who follows you are forgotten among the billions lost in the Cataclysm…"
```
Comment (NetSysFire) [inline data/json/snippets/epilogue_factions.json:8]:
```suggestion
        "text": "    You and everyone who followed you are forgotten among the billions lost in the Cataclysm…"
```

File: data/json/snippets/epilogue_factions.json [hunk 2]
```diff
@@ -3,10 +3,13 @@
...
+      },
       {
         "id": "epilogue_faction_your_followers_0",
-        "text": "    You are forgotten among the billions lost in the Cataclysm…"
+        "text": "    You and everyone who follows you are forgotten among the billions lost in the Cataclysm…"
```
Comment (NetSysFire) [inline data/json/snippets/epilogue_factions.json:12]:
```suggestion
        "text": "    You and everyone who followed you are forgotten among the billions lost in the Cataclysm…"
```

**Review by NetSysFire — COMMENTED** [review]
I think past tense is more appropriate here.

**Pull request merged**